### PR TITLE
11.0 pos qr scan and payment

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: pos_qr_payments/doc/changelog.rst [hunk 1]
```diff
@@ -1,3 +1,8 @@
+`1.0.1`
+-------
+
+- **FIX** - Error related to pos reference in different encodings
```
Comment (yelizariev) [inline pos_qr_payments/doc/changelog.rst]: The problem is in translation, not encoding

File: pos_qr_payments/models/pos_order.py [hunk 2]
```diff
@@ -11,4 +13,5 @@ class PosOrder(models.Model):
     @api.depends('pos_reference')
     def _compute_pos_reference_uid(self):
         for r in self:
-            r.pos_reference_uid = (r.pos_reference or ' ').split(' ', 1)[1]
+            reference = re.search('\d{5}\-\d{3}-\d{4}', r.pos_reference)
```
Comment (yelizariev) [inline pos_qr_payments/models/pos_order.py]: Amount of numbers is minimal, not exact value

File: pos_qr_scan/static/src/js/qr_scan.js [hunk 3]
```diff
@@ -128,7 +128,10 @@ odoo.define('pos_qr_scan', function(require){
                         self.stream = stream;
                         self.success(stream);
                     }).catch(function(error){
-                        error(error);
+                        self.gui.show_popup('error-traceback',{
+                            'title': error.name + ' ' + error.code,
+                            'body':  error.message
+                        });
```
Comment (yelizariev) [inline pos_qr_scan/static/src/js/qr_scan.js:134]: :+1:

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by yelizariev — APPROVED** [review]

**Comment by yelizariev** [comment]
Checks are red :disappointed: 
```

2018-10-18 15:50:51,739 4645 ERROR openerp_test odoo.addons.pos_wechat.tests.test_wechat_order: `   File "/home/travis/build/it-projects-llc/pos-addons/pos_qr_payments/models/pos_order.py", line 16, in _compute_pos_reference_uid
2018-10-18 15:50:51,740 4645 ERROR openerp_test odoo.addons.pos_wechat.tests.test_wechat_order: `     reference = re.search(u'\d{1,}-\d{1,}-\d{1,}', r.pos_reference)
2018-10-18 15:50:51,740 4645 ERROR openerp_test odoo.addons.pos_wechat.tests.test_wechat_order: `   File "/home/travis/virtualenv/python3.5.6/lib/python3.5/re.py", line 173, in search
2018-10-18 15:50:51,740 4645 ERROR openerp_test odoo.addons.pos_wechat.tests.test_wechat_order: `     return _compile(pattern, flags).search(string)
2018-10-18 15:50:51,740 4645 ERROR openerp_test odoo.addons.pos_wechat.tests.test_wechat_order: ` TypeError: expected string or bytes-like object
```

**Comment by KolushovAlexandr** [comment]
@ilmir-k Check it please

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Comment by yelizariev** [comment]
too red https://travis-ci.com/it-projects-llc/pos-addons/jobs/152935235

**Review by yelizariev — APPROVED** [review]

**Pull request merged**